apps/dot-voting/package.json

```diff
@@ -81,7 +92,7 @@
   "keywords": [],
   "license": "GPL-3.0",
   "lint-staged": {
-    "*.js": [
+    "app/**/*.js": [
```

Comment:
Might it be useful to also lint other `js` files outside the `app` folder? like for example the test files, linter configs and truffle-config, just asking, I would like to understand better the motivation for this linter scope limitation :)

Reply:
I did not intentionally add this; I think it came from #1154. I'll remove it!

Reply:
I think this lint setting is why some of those unused variables weren't detected by the linter! Great find, thank you!